docs/self-managed/installation-methods/helm/upgrade/helm-870-880.md

```diff
@@ -184,20 +205,13 @@ Enable or disable the required Camunda components for your use case:
 ##### 1.2.1 Enable Operate and Tasklist importers
 
 In 8.8, the Operate and Tasklist importers are disabled by default because they are not required for normal execution.  
-When upgrading from `8.7.x`, you must enable them to process any remaining data so that the upgrade can complete successfully.
-
-To enable the importers, set:
-
-- `camunda.operate.importerEnabled=true`
-- `camunda.tasklist.importerEnabled=true`
+When upgrading from `8.7.x`, they are enabled for the importer job only.
```

Comment:
When upgrading from `8.7.x`, Importers are enabled during the migration process for the importer job only. Once the cluster upgrade is complete, they can be disabled.